File: sharingCache/SampleData/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SampleData
{

    /// <summary>
    /// Model class for Customers
    /// </summary>
    [Serializable]
    public class Customer
    {
        public Customer()
        { }

        /// <summary>
        /// Unique Id of the customer
        /// </summary>
        public string CustomerID
        {
            get;
            set;
        }

        /// <summary>
        /// Contact name of the customer
        /// </summary>
        public virtual string ContactName
        {
            set;
            get;
        }

        /// <summary>
        /// Company the customer works for
        /// </summary>
        public virtual string CompanyName
        {
            set;
            get;
        }

        /// <summary>
        /// Contact number of the customer
        /// </summary>
        public virtual string ContactNo
        {
            set;
            get;
        }

        /// <summary>
        /// Residential address of the customer
        /// </summary>
        public virtual string Address
        {
            set;
            get;
        }

        /// <summary>
        /// Residence city of the customer
        /// </summary>
        public virtual string City
        {
            set;
            get;
        }

        /// <summary>
        /// Nationality of the customer
        /// </summary>
        public virtual string Country
        {
            set;
            get;
        }

        /// <summary>
        /// Postal code of the customer
        /// </summary>
        public virtual string PostalCode
        {
            set;
            get;
        }

        /// <summary>
        /// Fax number of the customer
        /// </summary>
        public virtual string Fax
        {
            set;
            get;
        }



    }
}

File: json-api-JAVA/src/main/java/org/example/Customer.java
package org.example;

import java.io.Serializable;

public class Customer implements Serializable
{
    private String CustomerID;
    private String ContactName;
    private String CompanyName;
    private String ContactNo;
    private String ContactTitle;
    private String Address;
    private String City;
    private String Country;
    private String region;
    private String phone;

    public String getContactTitle() {
        return ContactTitle;
    }

    public void setContactTitle(String contactTitle) {
        ContactTitle = contactTitle;
    }

    public String getRegion() {
        return region;
    }

    public void setRegion(String region) {
        this.region = region;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    private String postalCode;
    private String fax;

    public Customer()
    {
    }

    /**
     Unique Id of the customer
     */
    public String getCustomerID()
    {
        return CustomerID;
    }
    public void setCustomerID(String value)
    {
        CustomerID = value;
    }

    /**
     Contact name of the customer
     */
    public String getContactName()
    {
        return ContactName;
    }
    public void setContactName(String value)
    {
        ContactName = value;
    }

    /**
     Company the customer works for
     */
    public String getCompanyName()
    {
        return CompanyName;
    }
    public void setCompanyName(String value)
    {
        CompanyName = value;
    }

    /**
     Contact number of the customer
     */
    public String getContactNo()
    {
        return ContactNo;
    }
    public void setContactNo(String value)
    {
        ContactNo = value;
    }

    /**
     Residential address of the customer
     */
    public String getAddress()
    {
        return Address;
    }
    public void setAddress(String value)
    {
        Address = value;
    }

    /**
     Residence city of the customer
     */
    public String getCity()
    {
        return City;
    }
    public void setCity(String value)
    {
        City = value;
    }

    /**
     Nationality of the customer
     */
    public String getCountry()
    {
        return Country;
    }
    public void setCountry(String value)
    {
        Country = value;
    }

    /**
     Postal code of the customer
     */
    public String getPostalCode()
    {
        return postalCode;
    }
    public void setPostalCode(String value)
    {
        postalCode = value;
    }

    /**
     Fax number of the customer
     */
    public String getFax()
    {
        return fax;
    }
    public void setFax(String value)
    {
        fax = value;
    }

    @Override
    public String toString() {
        return "Customer{" +
                "CustomerID='" + CustomerID + '\'' +
                ", ContactName='" + ContactName + '\'' +
                ", CompanyName='" + CompanyName + '\'' +
                ", ContactNo='" + ContactNo + '\'' +
                ", ContactTitle='" + ContactTitle + '\'' +
                ", Address='" + Address + '\'' +
                ", City='" + City + '\'' +
                ", Country='" + Country + '\'' +
                ", region='" + region + '\'' +
                ", phone='" + phone + '\'' +
                ", postalCode='" + postalCode + '\'' +
                ", fax='" + fax + '\'' +
                '}';
    }
}
